src/components/Pages/Auth/AuthForm.tsx

```diff
@@ -0,0 +1,35 @@
+'use client'
+
+import Button from '@/components/UI/Button'
+import Form from '@/components/UI/Form/Form'
+import FormCheckbox from '@/components/UI/Form/FormCheckbox'
+import FormControl from '@/components/UI/Form/FormControl'
+import FormInput from '@/components/UI/Form/FormInput'
+import FormLabel from '@/components/UI/Form/FormLabel'
+import { FC, FormEvent } from 'react'
+
+const AuthForm: FC = () => {
+  return (
+    <Form
+      className="mt-5"
+      onSubmit={(e: FormEvent<HTMLFormElement>) => console.log(e.target)}
+    >
+      <FormControl>
+        <FormLabel htmlFor="email">
+          Confirm your email and get dynamically generated code
```

Comment:
We did not discuss it yet, but I think we should add i18n. You can skip it for this PR, and we will refactor later.

Reply:
Surely, for now it's just for testing purposes